country-map.js

```diff
@@ -139,6 +47,14 @@ module.exports = {
         "abbreviation": "AW",
         "currency": "AWG"
     },
+    "Ascension Island": {
+        "abbreviation": "AC",
+        "currency": "USD"

```

Comment:
Only made a change if >30% of our data from a given country came from it's currency - here 70% is USD, only 14% GBP.